src/map/npc.c

```diff
@@ -5723,8 +5723,10 @@ static bool npc_unloadfile(const char *filepath, bool unload_mobs)
 
 	dbi_destroy(iter);
 
-	if (found) /// Refresh event cache.
+	if (found) { /// Refresh event cache.
+		npc->motd = npc->name2id("HerculesMOTD");
```

Comment:
```suggestion
		npc->motd = npc->name2id("HerculesMOTD");
```
maybe consider using constant? or declare in /conf/script.conf ?  less hard-coded content.

Reply:
may be, but this is only crash fix